docs/src/App.vue

```diff
@@ -12,6 +12,15 @@ export default {
   name: 'App',
 
   setup () {
+    const $q = useQuasar()
+    console.log($q.localStorage.getItem('darkMode'))
+    if ($q.localStorage.getItem('darkMode') === null || $q.localStorage.getItem('darkMode') === true) {
```

Comment:
We never defined dark mode would be the default one, we'll discuss this Friday directly

In the meantime, this would be more readable

```js
$q.dark.set($q.localStorage.getItem('darkMode') ?? true)
```

Reply:
Ok. 

Scott